pkgs/by-name/fi/fishnet/package.nix

```diff
@@ -51,12 +51,12 @@ rustPlatform.buildRustPackage (finalAttrs: {
     versionCheckHook
   ];
   doInstallCheck = true;
-  versionCheckProgram = "${placeholder "out"}/bin/${finalAttrs.meta.mainProgram}";
+  versionCheckProgram = "${placeholder "out"}/bin/fishnet";
```

Comment:
You don't need inline it here. It is the default since https://github.com/NixOS/nixpkgs/commit/befb2beebd2382aa9a6274b0fd609fbd872d8cc8

Please keep current code or simply dropping entire this line.